network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt

```diff
@@ -89,4 +89,11 @@ enum class NetworkProtectionPixelNames(
     NETP_WAITLIST_NOTIFICATION_LAUNCHED("m_netp_ev_waitlist_notification_launched_c"),
     NETP_WAITLIST_BETA_ENABLED_DAILY("m_netp_ev_waitlist_enabled_d"),
     NETP_WAITLIST_BETA_ENABLED("m_netp_ev_waitlist_enabled_c"),
+    NETP_GEOSWITCHING_PAGE_SHOWN("m_netp_imp_geoswitching_c", enqueue = true),
```

Comment:
Enqueueing because NetP might still be connecting when user opens the page and pixel might fail to send.